contracts/modules/LoanSettings/LoanSettings.sol

```diff
@@ -93,6 +95,99 @@ contract LoanSettings is State, InterestHandler, LoanSettingsEvents {
         }
     }
 
+    function migrateLoanParamsList(
+        address owner,
+        uint256 start,
+        uint256 count)
+        external
+        onlyGuardian
+    {
+        EnumerableBytes32Set.Bytes32Set storage set = userLoanParamSets[owner];
+        uint256 end = start.add(count).min256(set.length());
+        if (start >= end) {
+            return;
+        }
+        count = end-start;
+
+        bytes32 loanParamId;
+        LoanParams memory loanParamsLocal;
+
+        for (uint256 i = start; i < end; ++i) {
+            loanParamsLocal = loanParams[set.get(i)];
+            loanParamId = getLoanParam(
+                loanParamsLocal.loanToken,
+                loanParamsLocal.collateralToken,
+                loanParamsLocal.maxLoanTerm == 0 // isTorqueLoan
```

Comment:
isnt maxLoanTerm always equal to 0 as they are infinite term loans?

Reply:
so before it was used to indicate whether its fulcrum or torque. if zero means Torque loan other wise would indicate the length for Fulcrum loans( 28 days then rollover)

Reply:
ya but today that is not the case so it should be reworked a bit.

Reply:
so we can use either 0 or 1 to represent margin or borrow

Reply:
ya that is good